sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md

```diff
@@ -1,6 +1,6 @@
 # Guide for migrating to Azure.Messaging.ServiceBus from Microsoft.Azure.ServiceBus
 
-This guide is intended to assist in the migration to version 7 of the Service Bus client library [`Azure.Messaging.ServiceBus`](https://www.nuget.org/packages/Azure.Messaging.ServiceBus/) from version 4 of [`Microsoft.Azure.ServiceBus`](https://www.nuget.org/packages/Microsoft.Azure.ServiceBus/). It will focus on side-by-side comparisons for similar operations between the two packages. 
+This guide is intended to assist in the migration to version 7 of the Service Bus client library [`Azure.Messaging.ServiceBus`](https://www.nuget.org/packages/Azure.Messaging.ServiceBus/) from [`Microsoft.Azure.ServiceBus`](https://www.nuget.org/packages/Microsoft.Azure.ServiceBus/). It will focus on side-by-side comparisons for similar operations between the two packages. 
```

Comment:
Also update https://github.com/Azure/azure-sdk-for-net/pull/21041/files#diff-5d66174561b8ab860faf51d85b17200f8200ee26e316f65188a3136ce487ca30R53

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-net/pull/21041/commits/554349c411852bc9c9c0f482ad32e8dcdff605d5